core/src/main/java/brooklyn/management/ha/HighAvailabilityManagerImpl.java

```diff
@@ -1077,4 +1077,13 @@ public String toString() {
         return result;
     }
     
+    @Override
+    public long getLastStateChange() {
+        if (nodeStateHistory.size() > 0) {
+            return (Long)nodeStateHistory.get(0).get("timestamp");
+        } else {
+            return 0;
```

Comment:
Personal preference for returning `-1` to signify not available (as 0 is an unlikely but valid timestamp)